modules/monitoring-choosing-a-metrics-collection-profile.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="choosing-a-metrics-collection-profile_{context}"]
+= Choosing a metrics collection profile
+
+To choose a metrics collection profile for core {product-title} monitoring components, edit the `cluster-monitoring-config` `ConfigMap` object.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+* You have enabled Technology Preview features by using the `FeatureGate` custom resource (CR).
```

Comment:
```suggestion
* You have enabled Technology Preview features by using the `FeatureGate` custom resource (CR).
* You have created the `cluster-monitoring-config` `ConfigMap` object.
```
change the order, since enable TechPreview takes some minutes to wait for all nodes become ready

Reply:
Changed per your suggestion